### ISSUE TITLE
barWidth not working

### ISSUE THREAD

**Issue by ashupanwar** [tiagomotasantos/react-audio-visualizers#45]
only a few props are working like numBars and color etc, but setting the value to barWidth does not do anything.

**Comment by Bodrie** [comment]
Agree, its because its not getting the value, its getting only the const value, you can see it in the source files.
